src/core/mac/mac_types.hpp

```diff
@@ -424,10 +416,7 @@ OT_TOOL_PACKED_BEGIN
 class Key : public otMacKey, public Equatable<Key>, public Clearable<Key>
 {
 public:
-    enum
-    {
-        kSize = OT_MAC_KEY_SIZE, // Key size in bytes.
-    };
+    static constexpr uint16_t kSize = OT_MAC_KEY_SIZE; // Key size in bytes.
```

Comment:
Is it intentional not using `//<`? What practice are we going to take for `//` and `//<`?

Reply:
No. Not intentional. I probably just did not notice it and left it as `//` which was before. I will change it in next push. Thanks.

In general, here is what I would recommend about this:
- We use `///<` only for `public` definitions (we can also skip it if the constant name is descriptive enough and comment does not add much value.). 
- We use `///<` only when the entire description fits in same line, otherwise it's better to have a comment before the definition using `/** */` style.
- For `private` definitions, adding comment/docs is fully optional (up to developer), but we should not use `///<` so it won't be added in doxygen (or to other tools that collect such docs) since the definitions is private.